.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,48 @@
+name: Serialize snacks 
+
+on:
+  pull_request:
+
+jobs:
+  core-build:
+    name: Build Core example
```

Comment:
nit : instead of core this is actually building all examples which  is good to catch issues earlier

Reply:
no, it's only a building core example.